modules/Cargo.toml

```diff
@@ -37,6 +37,8 @@ dyn-clonable = "0.9.0"
 regex = "1"
 bech32 = "0.7.2"
 
+eyre = "0.6.5"
```

Comment:
This should be moved the `dev-dependencies` section as it is only used in tests and we want to keep the `modules` crate free of error handling libraries.

Reply:
Done in d57cf4119a9dbfe5260ef7c1afc212bc5c305458. Once `modelator.rs` lives in its own crate, we will depend only on that, so that dependency can go away.